docsets/Terraform/docset.json

```diff
@@ -12,6 +12,10 @@
         "terra"
     ],
     "specific_versions": [
+        {
+            "version": "1.2.4.220702",
+            "archive": "versions/1.2.4.220702/Terraform.tgz"
```

Comment:
This file does not exist. The file you added is at: `versions/v1.2.4.220702/Terraform.tgz` (notice the `v` before the version).

Reply:
Thanks; that's now been fixed.